src/IntercomClient.php

```diff
@@ -58,6 +58,16 @@ class IntercomClient
      */
     public $users;
 
+    /**
+     * @var IntercomContacts $contacts
+     */
+    public $contacts;
+
+    /**
+     * @var IntercomCustomers $customers
+     */
+    //public $customers;
```

Comment:
I know this is work in progress, but wanted to add a reminder that this and its initializer are commented out 😄